staging/src/k8s.io/client-go/tools/record/event_test.go

```diff
@@ -631,15 +641,18 @@ func TestMultiSinkCache(t *testing.T) {
 		},
 	}
 	testPod2 := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
You are right. The test should assume TypeMeta is empty in memory. Now i reverted the addition and changed to use scheme.Scheme.